dask_geomodeling/raster/elemwise.py

```diff
@@ -75,20 +87,17 @@ def timedelta(self):
         """ The period between timesteps in case of equidistant time. """
         if len(self._sources) == 1:
             return self._sources[0].timedelta
-
         timedeltas = [s.timedelta for s in self._sources]
-        if any(timedelta is None for timedelta in timedeltas):
-            return None  # None precedes
-
-        # multiple timedeltas: assert that they are equal
-        if not timedeltas[1:] == timedeltas[:-1]:
-            raise ValueError(
-                "Time resolutions of input rasters are not "
-                "equal ({}).".format(timedeltas)
-            )
+        if any(x is None for x in timedeltas):
+            return None
         else:
             return timedeltas[0]
 
+    @property
+    def temporal(self):
+        """If any of the sources is temporal, the result is temporal."""
```

Comment:
This is enforced by the __init__(), but maybe not the most adequate description for this property.